### PR TITLE
ENDOC-90 Rework/consolidate page and page templates docs

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
This is visible on my GH Pages currently - https://nshaw.github.io/next/tutorials/cms/page-management.html. I consolidated a number of page/page template docs, renamed the section Content Management to Pages and Content since it had quite a bit more than just Content Management, and updated language in most of those docs to be more consistent.

File: vuepress/docs/next/tutorials/cms/page-management.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    page template you associated with that page.
+
+![image](./extracted-media/media/Publish3.png)
+
+-   Add widgets by dragging them from the right column on to the desired frame. 
+-   If you select the **Content** widget, its corresponding settings page will be shown.
+
+![image](./extracted-media/media/Publish4.png)
+
+-   Click **Add existing content** to bring up the Content selection view.
```
Comment (joewhite101) [inline vuepress/docs/next/tutorials/cms/page-management.md:40]: Looking at the screenshot on this one I wonder if we should explain the colors in the status column for the content. Or grab a new screenshot with all yellow content. The yellow dot stands out.

**Review by joewhite101 — CHANGES_REQUESTED** [review]
One small question

File: vuepress/docs/next/tutorials/cms/page-management.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/cms/page-management.md:40]: Good catch.

**Comment by nshaw** [comment]
@joewhite101 https://nshaw.github.io/next/tutorials/cms/page-management.html#create-a-page is updated with the latest.

**Comment by joewhite101** [comment]
LGTM

**Review by joewhite101 — APPROVED** [review]
LGTM

**Pull request merged**